### PR TITLE
Adds AdventureLog

### PR DESCRIPTION
**Pull request by seanmorley15**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/adventurelog.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+demo_url: "https://adventurelog.app"
+```
+
+To ensure your issue is dealt with swiftly, please check the following (check the boxes `[x]`):
+- [x] Submit one item per pull issue. This eases reviewing and speeds up inclusion.
+- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
+- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
+- [x] Any software project you are adding to the list is actively maintained.
+- [x] Any software project you are adding was first released more than 4 months ago.
+- [x] Any software project you are adding has working installation instructions.
```
Comment (Ki-er) [inline software/adventurelog.yml:30]:
```suggestion
```

File: software/adventurelog.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+name: Addition
+about: Add new software to the list.
+title: Add AdventureLog
+labels: addition
+assignees: ''
+
+---
+```yaml
```
Comment (Ki-er) [inline software/adventurelog.yml:9]:
```suggestion
```

**Review by Ki-er — CHANGES_REQUESTED** [review]

File: software/adventurelog.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+website_url: "https://adventurelog.app"
+source_code_url: "https://github.com/seanmorley15/AdventureLog"
+description: "Self-hostable travel tracker and trip planner."
+licenses:
+  - GPL-3.0
+platforms:
+  - Docker
+tags:
+  - Maps and Global Positioning System (GPS)
+depends_3rdparty: false
```
Comment (Ki-er) [inline software/adventurelog.yml:11]:
```suggestion
```

File: software/adventurelog.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+name: "AdventureLog"
+website_url: "https://adventurelog.app"
+source_code_url: "https://github.com/seanmorley15/AdventureLog"
+description: "Self-hostable travel tracker and trip planner."
```
Comment (Ki-er) [inline software/adventurelog.yml:4]:
```suggestion
description: "Travel tracker and trip planner."
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Couple more changes

**Review by nodiscc — APPROVED** [review]

**Pull request merged**